guides/uploading_patient_attachments.md

```diff
@@ -55,7 +55,9 @@ To do so, make a POST request to the patient attachments endpoint, with the pati
 `https://api.cliniko.com/v1/patient_attachments`
 ```json
 {
-  "patient_id": "382",
+  "description": "Custom description for this attachment",
+  "filename": "The name of the file.txt",
```

Comment:
I don't think we allow setting this do we?

Reply:
I have been setting description via this method, which is a very useful function.

Reply:
Gah - will remove `filename`